homeassistant/helpers/helper_config_entry_flow.py

```diff
@@ -337,3 +366,22 @@ def wrapped_entity_config_entry_title(
     if state:
         return state.name or object_id
     return object_id
+
+
+@callback
+def exclude_own_entities(
```

Comment:
Let's give it a better name. What about `entity_selector_without_helper_entities`

Reply:
Fixed in cc8a589